stock_inventory_valuation_report/wizard/stock_quantity_history.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.tools.safe_eval import safe_eval
+
+
+class StockQuantityHistory(models.TransientModel):
```

Comment:
Here too. Please add docstrings for methods.

Reply:
I added some docstrings, but I don't find it very helpful to add a docstring "Produces the HTML report" to a method that's already called "button_export_html".

Doesn't that just create more noise?